settings.gradle

```diff
@@ -18,6 +18,8 @@ enableFeaturePreview('GRADLE_METADATA')
 
 ext.native_targets_enabled = properties['disable_native_targets'] == null
 
+apply from: file('gradle/versions.gradle')
```

Comment:
It will affect configuration perf drammatically

Reply:
Why? Is including a file so slow?